src/contracts/evm/AccountDepositWithdrawEndowments/abi.json

```diff
@@ -0,0 +1,4 @@
+[
+  "event UpdateEndowmentState(uint256,tuple(tuple(uint256,uint256),tuple(tuple(uint256,uint256[],address[]),tuple(uint256,uint256[],address[])),bool,tuple(tuple(uint256,address),uint8)))",
```

Comment:
fragment `event UpdateEndowmentState` could just be added to Accounts abi as this is facet (we don't interact with this directly) of that contract

Reply:
Makes sense